### PR TITLE
Add http -> https redirect on the ALB

### PR DESCRIPTION
**Pull request by waeltken**
Solves #24.

Creates a new security group to allow http ingress to the ALB and a new lb listener rule to redirect traffic from port 80 HTTP to 443 HTTPS.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -135,6 +161,19 @@ module "alb_https_sg" {
   tags = "${local.tags}"
 }
 
+module "alb_http_sg" {
+  source  = "terraform-aws-modules/security-group/aws//modules/http-80"
+  version = "v2.9.0"
+
+  name        = "${var.name}-alb-http"
+  vpc_id      = "${local.vpc_id}"
+  description = "Security group with HTTP ports open for everybody (IPv4 CIDR), egress ports are all world open"
```
Comment (antonbabenko) [inline main.tf]: `open for everybody` in the description should be changed to reflect the reality.

Comment (waeltken) [reply main.tf]: Well... it's as open as the HTTPS security group to allow to respond to any request with the redirect to HTTPS.

Comment (antonbabenko) [reply main.tf]: Yes, I understand. I thought it should be more correct like this - "Security group with HTTP ports open for specific IPv4 CIDR block (or everybody), egress ports are all world open". Same for `https`.

**Comment by antonbabenko** [comment]
Thanks @waeltken !

[v1.9.0](https://github.com/terraform-aws-modules/terraform-aws-atlantis/releases/v1.9.0) has been just released.

**Pull request merged**